test/lib/ansible_test/_data/completion/docker.txt

```diff
@@ -1,4 +1,5 @@
-default name=quay.io/ansible/default-test-container:1.14 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined
+default name=quay.io/ansible/default-test-container:2.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=collection
```

Comment:
I feel like `context` may not be required here but the context should be based on where this file is located. For example ones in this file should be "global" defaults but a collection/base can override them in `test(s)/completion/docket.txt`.

This allows `ansible-test` to define the default `default` docker container image but also allow base, and a mechanism for other collections to override it with their own one.

Reply:
I'm planning on building a customization system for collections to define workflow, settings, etc. for testing in 2.11. It will include support for specifying custom containers.

For 2.10 I'd like to limit the change to splitting the container settings between ansible-base and collections, rather than trying to quickly throw together something that will likely be replaced in 2.11.

Reply:
The trouble is once you've defined the `context=...` part in are release you are sort of stuck with it. I just don't see how this could be used in future version without people repeating themselves, i.e. `context=<namespace>.<collection>` when ansible-test can derive that information itself.

Reply:
Why are we stuck with it? It's an internal data file for ansible-test. Users can't change it or build their own, so we're free to do whatever we want with the file, including removing it or changing the format.

Reply:
That is true, sorry I was thinking this was a public file like we have with the sanity tests.